CMakeLists.txt

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-project (sxg)
+cmake_policy(SET CMP0048 NEW)
```

Comment:
Is this change relevant (needed to make this Cmake generate *.so)?

Reply:
Oh, not required.